src/sagas/index.js

```diff
@@ -113,6 +113,7 @@ const VM_FETCH_ADDITIONAL_DEEP = [
   'sessions',
   'snapshots',
   'statistics',
+  'nics.reporteddevices',
```

Comment:
No need to specify the nics twice.

A single `'nics.reported_devices'` is preferred.